assets/constants.ts

```diff
@@ -6872,7 +6872,7 @@ export default {
 
   "FAVICON_ALERT_PATH": "/assets/images/favicon_alert/favicon_alert.ico",
 
-  "CAN_FETCH_VERSION_HISTORY_DATA": false,
+  "ALLOW_VERSION_HISTORY_EXPLORATION": false,
```

Comment:
I don't understand what this means. Could you rename it to be clearer?

Reply:
It was to decide whether we should allow the frontend to fetch version history data from the backend. I have removed it now and we don't need it anymore.